src/services/impressions/dto.js

```diff
@@ -36,6 +36,10 @@ module.exports = {
           if (entry.label) keyImpression.label = entry.label;
           if (entry.bucketingKey) keyImpression.bucketingKey = entry.bucketingKey;
 
+          properties = {};
```

Comment:
Please add the `const` keyword. In JavaScript, declaring a variable without any keyword, would result in a global variable.
You could use const as this properties variable will always contain that object.